app/controllers/users/phone_setup_controller.rb

```diff
@@ -35,9 +35,11 @@ def create
 
       if result.success?
         handle_create_success(@new_phone_form.phone)
-      elsif recoverable_recaptcha_error?(result)
-        render :spam_protection
       else
+        if result.errors.key?(:recaptcha_token)
+          flash.now[:error] = result.errors[:recaptcha_token].first
```

Comment:
Would `FormResponse#first_error_message` work here? Or is `:recaptcha_token` not the only key?

Reply:
I thought about `first_error_message`, but there are other fields which may have validation errors, and for now we'd only want to show the error banner for reCAPTCHA validation.

Reply:
another idea for the future might be add an optoinal `key` arg to `first_error_message` it could be
```
form.first_error_message(:recaptcha_token)
```

Reply:
That's an interesting idea. One of the other approaches I'd considered here was doing something like `flash.now[:error] = result.errors[:recaptcha_token]&.first` and dropping the condition altogether, but aside from readability, I was concerned about overriding the flash if it had been previously assigned and there wasn't something to replace it with (`nil`). Something like that could also happen with your suggestion.

```rb
flash[:error] = '...'

# ...

flash[:error] = form.first_error_message(:recaptcha_token) # overridden to `nil` if no reCAPTCHA errors
```

I suppose that's probably not very likely to happen since the controller action would probably be the only place we'd typically expect it to be assigned.

Reply:
Gave it a go in 77c71230f6 and eab2d6bfcf to support and use a parameterized `first_error_message`.

One non-obvious change here is that I dropped the `Array` coercion. I don't know that we ever initialize `FormResponse` with errors given in anything other than an array of strings?